src/components/Top/Banner.tsx

```diff
@@ -0,0 +1,43 @@
+import { useRouter } from "next/router";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+const Banner = () => {
+  const router = useRouter();
+  const {
+    query: { year },
+  } = router;
+
+  return (
+    <div className="banner">
+      <div className="container">
+        <h1>
+          <FormattedMessage id="header.title" />
+        </h1>
+      </div>
+      <style jsx>{`
+        .banner {
```

Comment:
I'm not sure introducing `styled-jsx` is the best option, I wonder if we should introduce a solution like Emotion, but since `styled-jsx` is nativelt supported by Next.js, it was a simple solution.